src/test/java/com/ss/ita/google/OpenSearchResultPageNineTest.java

```diff
@@ -5,15 +5,14 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-public class OpenResultPageNineTest extends TestRunner {
+public class OpenSearchResultPageNineTest extends TestRunner {
 
     @Test
     public void verifyOpenPageNine() {
         int pageNumber = 9;
-        int currentPageNumber = new HomePage()
+        new HomePage()
                 .doSearch("funny kitten")
-                .openResultPage(pageNumber)
-                .getCurrentPageNumber();
-        Assert.assertEquals(currentPageNumber, pageNumber);
+                .openSearchResultPageNumber(pageNumber)
+                .shouldHavePageNumber(pageNumber);
```

Comment:
dont do it like method shouldHavePageNumber(pageNumber). Do it like assertion in test.